src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowModuleBuilding.java

```diff
@@ -24,7 +24,7 @@
 public abstract class AbstractWindowModuleBuilding<B extends IBuildingView> extends AbstractModuleWindow
 {
     /**
-     * Type B is a class that extends {@link com.minecolonies.coremod.colony.buildings.AbstractBuildingWorker.View}.
+     * Type B is a class that extends {@link com.minecolonies.coremod.colony.buildings.AbstractBuildingWorkerView}.
```

Comment:
Isn't this file at com.minecolonies.coremod.colony.buildings.views.AbstractBuildingWorkerView?